### PR TITLE
[Test/C-Api] add testcases

### PR DESCRIPTION
**Pull request by jaeyun-jung**
migrate c-api testcases from nnstreamer repo.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: packaging/machine-learning-api.spec [hunk 1]
```diff
@@ -139,6 +194,12 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir
 
 ninja -C build %{?_smp_mflags}
 
+# Run test
+%if 0%{?unit_test}
+bash %{test_script} ./tests/capi/unittest_capi_inference
+bash %{test_script} ./tests/capi/unittest_capi_inference_nnfw_runtime
+%endif
```
Comment (gichan-jang) [inline packaging/machine-learning-api.spec:201]: Don't run `unittest_capi_inference_latency`?

Comment (jaeyun-jung) [reply packaging/machine-learning-api.spec:201]: unittest_capi_inference_latency is benchmark tool, to measure invoke time.
It contains single-shot invoke function with imgclf model and this is already in unittest_capi_inference.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by helloahn — APPROVED** [review]
LGTM

**Comment by jaeyun-jung** [comment]
2021-02-19 PR updated (add new pkg for unittest)

**Pull request merged**